src/features/events/models/EventDataModel.ts

```diff
@@ -213,6 +225,8 @@ export default class EventDataModel extends ModelBase {
 
       if (startTime > now) {
         return EventState.SCHEDULED;
+      } else if (data.cancelled) {
+        return EventState.CANCELLED;
```

Comment:
This logic is not working as I would expect. If an event is cancelled, everything else should be irrelevant, so I think this if statement needs to be at the very top, i.e. above line 222.

Reply:
Of course! I'll change it asap!